test/index.test.js

```diff
@@ -321,4 +343,36 @@ describe('module-resolver', () => {
             );
     });
   });
+
+  describe('no babelrc in path', () => {
+    const transformerOpts = {
+      babelrc: false,
+      plugins: [
+        [plugin, {
+          cwd: 'babelrc',
+        }],
+      ],
+    };
+    const { cwd } = process;
+
+    function mockCwd() {
+      return '/.fake';
+    }
+
+    beforeEach(() => {
+      process.cwd = mockCwd;
+    });
+
+    afterEach(() => {
+      process.cwd = cwd;
+    });
+
+    describe('should leave the path as is', () => {
+      testRequireImport(
+                './test/path',
```

Comment:
Shouldn't this not start with `./`? Otherwise, because we'll not alias anything anyway when the path is already relative